### PR TITLE
wallet use new transferObject method

### PR DESCRIPTION
**Pull request by Jibz1**
Update transfer NFT to use `transferObject` metho

https://user-images.githubusercontent.com/8676844/176786872-04be23a6-23b9-4a2a-8e48-f301ff49fbbc.mov

d

### CODE REVIEW

File: wallet/src/ui/app/pages/transfer-nft/TransferNFTForm.tsx [hunk 1]
```diff
@@ -31,10 +29,10 @@ function TransferNFTForm({
         isValid,
         values: { to, amount },
     } = useFormikContext<FormValues>();
-    const intl = useIntl();
+
     const onClearRef = useRef(onClearSubmitError);
     onClearRef.current = onClearSubmitError;
-    const transferCost = 10000;
+    const transferCost = 450;
```
Comment (666lcz) [inline wallet/src/ui/app/pages/transfer-nft/TransferNFTForm.tsx:35]:
```suggestion
    const transferCost = 430;
```

**Review by gegaowp — COMMENTED** [review]
thanks for working on this! 
the codes look good overall, can you do a quick video or screenshots?

File: wallet/src/ui/app/pages/transfer-nft/TransferNFTForm.tsx [hunk 2]
```diff
@@ -52,29 +50,8 @@ function TransferNFTForm({
...
-                        BigInt(gasBalance),
-                        balanceFormatOptions
-                    )}{' '}
-                </div>
-                <ErrorMessage
-                    className={st.error}
-                    name="amount"
-                    component="div"
-                />
+                * Total transaction fee estimate (gas cost): 430 {GAS_SYMBOL}
```
Comment (666lcz) [inline wallet/src/ui/app/pages/transfer-nft/TransferNFTForm.tsx:53]: use `transferCost`

File: wallet/src/ui/app/redux/slices/sui-objects/NFT.ts [hunk 3]
```diff
@@ -38,13 +38,10 @@ export class ExampleNFT {
...
-        return await signer.executeMoveCall({
-            packageObjectId: '0x2',
-            module: 'devnet_nft',
-            function: 'transfer',
-            typeArguments: [],
-            arguments: [nftId, recipientID],
-            gasBudget: 10000,
+        return await signer.transferObject({
+            objectId: nftId,
+            gasBudget: 430,
```
Comment (666lcz) [inline wallet/src/ui/app/redux/slices/sui-objects/NFT.ts:43]: use `transferCost` here

**Review by 666lcz — CHANGES_REQUESTED** [review]
overall looks good. Just a few nits to reuse constant

File: wallet/src/ui/app/pages/transfer-nft/TransferNFTForm.tsx (hunk 2 above)
Comment (666lcz) [reply wallet/src/ui/app/pages/transfer-nft/TransferNFTForm.tsx:54]: I think we are missing one here. We should export `transferCost`(nit: we should rename it to `DEFAULT_NFT_TRANSFER_GAS_FEE`) if it is defined in another file

**Review by 666lcz — APPROVED** [review]

**Pull request merged**